documentation-tree/react/saga.md

```diff
@@ -2,10 +2,96 @@
 
 ## General advice
 
-If there is logic in your saga, this logic should be isolated in a pure function that is tested. Do not test the saga itself.
+~~If there is logic in your saga, this logic should be isolated in a pure function that is tested. Do not test the saga itself.~~
```

Comment:
Why cross this out?

Reply:
You should test the sagas, and many people have this view I crossed out